nsqd/client_v2.go

```diff
@@ -343,6 +362,17 @@ func (c *clientV2) SendingMessage() {
 	atomic.AddUint64(&c.MessageCount, 1)
 }
 
+func (c *clientV2) PublishedMessage(topic string, count uint64) {
+	c.metaLock.Lock()
+	counter, ok := c.pubCounts[topic]
+	if ok {
```

Comment:
looking up a missing key in a golang map returns the zero value ... it would work, but would it be bad style, to just:

```go
c.pubCounts[topic] += count
```

Reply:
SGTM